meetings/2023-06-21.md

```diff
@@ -0,0 +1,130 @@
+# W3C Solid Community Group: Weekly
+
+* Date: 2023-06-21T14:00:00Z
+* Call: https://meet.jit.si/solid-cg
+* Chat: https://gitter.im/solid/specification
+* Repository: https://github.com/solid/specification
+* Status: Draft
+
+## Present
+* [Sarven Capadisli](https://csarven.ca/#i)
+* [elf Pavlik](https://elf-pavlik.hackers4peace.net)
+* [Pierre-Antoine Champin](https://solid.champin.net/pa/profile/card#me)
+* Laurens Debackere
+* Hadrian Zbarcea
+* [Jesse Wright (jeswr)](https://www.jeswr.org/#me)
+* Maxime Lecoq
+* April Daly
+* Jeff Zucker
+* [Rahul Gupta](https://cxres.pages.dev/profile#i)
+---
+
+## Announcements
+
+### Meeting Guidelines
+* [W3C Solid Community Group Calendar](https://www.w3.org/groups/cg/solid/calendar).
+* [W3C Solid Community Group Meeting Guidelines](https://github.com/solid/specification/blob/main/meetings/README.md).
+* No audio or video recording, or automated transcripts without consent. Meetings are transcribed and made public. If consent is withheld by anyone, recording/retention must not occur.
+* Join queue to talk.
+* Topics can be proposed at the bottom of the agenda to be discussed as time allows. Make it known if a topic is urgent or cannot be postponed.
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/).
+* [Solid Code of Conduct](https://github.com/solid/process/blob/main/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/)
+* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+* Pierre-Antoine
+
+### Introductions
+* name: text
+
+---
+
+
+## Topics
+
+### Recording demos
+* SC: I don't mind recording it but this raises the question of where to host the video afterward. I could put it on my website but the CG probably does not want to rely on my website being up -- and I don't want this responsibility.
```

Comment:
I missed the meeting, so technically this comment doesn't exist, but...

The obvious thought for video uploads is YouTube. I don't know current bandwidth or volume limitations on github.io, drive.google.com, dropbox.com, box.com, etc., but those seems like other possible places. For the latter bunch, I would think it OK to force download for local playback, as opposed to streaming video, which might lower the pipe if not disk consumption.

Reply:
Not sure I understand your reference to "technically this comment doesn't exist".. but I did say what's in the minutes in the meeting. If you meant your comment above, we briefly / informally? touched on that.

What's not captured in detail / the point was that it is CG's asset and that it should be controlled by the CG preferably under solidproject.org or solidcommunity.net. While some of those third-party services (github included) may be convenient to some - IMO - we/CG shouldn't rely on them (if anything but the negative impression it gives).

https://github.com/solid/solidproject.org/issues/248

Reply:
I meant, technically [_my_ comment](https://github.com/solid/specification/pull/535#discussion_r1237827765) doesn't exist.

In reply to [your comment](https://github.com/solid/specification/pull/535#discussion_r1241298254) here, "under solidproject.org or solidcommunity.net" only covers a couple of URL authority segments, which could lead *anywhere* for the final resource. So far as I'm aware, the CG doesn't have any bank account, server hardware, or other material assets with which to "control" (or host) assets such as demo videos; thus, third-party hosting services are worth consideration (at least).